easygui/boxes/choice_box.py

```diff
@@ -239,8 +233,9 @@ def get_pos(self):
         global_state.window_position = '+' + geom.split('+', 1)[1]
 
     def preselect_choice(self, preselect):
-        self.choiceboxWidget.select_set(preselect)
-        self.choiceboxWidget.activate(preselect)
+        if preselect <> None:
```

Comment:
I'm not sure which is better.  i typically use:
if preselect is not None:

I likely will switch it over to that, but generally just want to follow conventions where I can.  Let me know if you don't like my change.